install_config/aggregate_logging.adoc

```diff
@@ -12,14 +12,17 @@ toc::[]
 
 == Overview
 
-OpenShift cluster administrators can aggregate logs and enable application
-developers to view them for a range of OpenShift services using the EFK stack,
-which is a modified version of the
-https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. The EFK
-stack can be useful for viewing logs aggregated from hosts and applications,
+As an OpenShift cluster administrator, you can deploy the EFK stack to aggregate
+logs for a range of OpenShift services, and allow application developers to view
+them. The EFK stack is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. Once
+deployed, the stack communicates securely between the stack components by
+generating its own secrets.
```

Comment:
Fine up until the last phrase. The secrets are just a conveyance for what the components use to communicate securely. How to say this without using too much geek-speak... something like:

"Once deployed, the stack components communicate securely and log access is limited by the projects a user can view."